openedx/core/djangoapps/password_policy/constants.py

```diff
@@ -0,0 +1 @@
+PASSWORD_POLICY_COMPLIANT_USERS_GROUP_NAME = 'pp.compliant.users'
```

Comment:
if there is no performance value in a shorter name, prefer the full name `password_policy` to the acronym `pp`